pyproject.toml

```diff
@@ -1,96 +0,0 @@
-[build-system]
-requires = [
-    "setuptools>=45",
-    "setuptools_scm[toml]>=7.0",
-]
-build-backend = "setuptools.build_meta"
-
-[project]
-name = "ScippUncertainty"
-description = "Advanced uncertainty propagation with Scipp"
-license = { "file" = "LICENSE" }
-readme = "README.md"
-keywords = []
-classifiers = [
-    "Intended Audience :: Science/Research",
-    "License :: OSI Approved :: BSD License",
-    "Natural Language :: English",
-    "Operating System :: OS Independent",
-    "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3 :: Only",
-    "Programming Language :: Python :: 3.8",
-    "Programming Language :: Python :: 3.9",
-    "Programming Language :: Python :: 3.10",
-    "Programming Language :: Python :: 3.11",
-    "Topic :: Scientific/Engineering",
-    "Typing :: Typed",
-]
-dynamic = ["version"]
-requires-python = ">=3.8"
-dependencies = [
-    "numpy>=1.20",
-    "scipp"
-]
-
-[project.optional-dependencies]
-all = ["rich"]
-
-[project.urls]
-"Documentation" = "https://scipp.github.io/scipp-uncertainty"
```

Comment:
Yes. I'll rename the repo when the reviews are done. Hope there are no other uses of the name with hyphen.

Reply:
So the repo will have a hyphen, but not the package (obviously)?

Reply:
Actually, the repo doesn't even have a hyphen. Only these urls have it. Changed them.

This happened because initially, the repo had a hyphen but I had already renamed it.